examples/test-animate-color/index.html

```diff
@@ -0,0 +1,153 @@
+      </a-entity>
+
+      <a-entity position="0 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-0.5 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-1 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-1.5 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-2 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-2.5 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-3 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-3.5 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-4 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+      <a-entity position="-4.5 1 -3.5" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-animation attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-animation>
+      </a-entity>
+
+      <a-plane rotation="-90 0 0" width="40" height="40" color="#7BC8A4"></a-plane>
+    </a-scene>
+  </body>
+
+  <script>
+    window.onload = function() {
```

Comment:
Our linter isn't running on examples yet, but:

- Space after `function` (`function ()`)
- Space around operators (`length / 2`)

Might be better to wait for `scene` `loaded` event too.